logstash-core/lib/logstash/bootstrap_check/persisted_queue_config.rb

```diff
@@ -0,0 +1,22 @@
+# encoding: utf-8
+require "java"
+require "logstash/errors"
+
+module LogStash module BootstrapCheck
+    class PersistedQueueConfig
+      def self.check(settings)
+        if settings.get("queue.type") == "persisted"
```

Comment:
this is really a styling preference but I personally like to do a quick `return` to save an indentation. I find it ok to do such a "short-circuit" when it is the first(s) statement. For example:
```ruby
def self.check(settings)
  return unless settings.get("queue.type") == "persisted"
  ...
end
```